bridges/ARDMediathekBridge.php

```diff
@@ -0,0 +1,41 @@
+<?php
+class ARDMediathekBridge extends BridgeAbstract {
+	const NAME = 'ARD-Mediathek Bridge';
+	const URI = 'https://www.ardmediathek.de';
+	const DESCRIPTION = 'Feed of any series in the ARD-Mediathek, specified by its URI';
+	const MAINTAINER = 'yue_dongchen';
+
+	const PARAMETERS = array(
+		array(
+			'uri' => array(
+				'name' => 'URI',
+				'required' => true,
+				'defaultValue' => '45-min/Y3JpZDovL25kci5kZS8xMzkx/'
+			)
+		)
+	);
+
+	public function getURI() {
+		if(!is_null($uri = $this->getInput('uri')))
+			return 'https://www.ardmediathek.de/sendung/' . $uri;
+
+		return parent::getURI();
+	}
+
+	public function collectData() {
+		$html = getSimpleHTMLDOM($this->getURI());
+		if(!$html)
+			returnServerError('No response for' . $this->getURI() . '!');
```

Comment:
getSimpleHTMLDOM already handles cases, where server responses with non-200 http code.
Does ARM-Mediathek often returns empty html with 200 http code?

Reply:
I didn't know that. Thank you for point out the function! 

In that case, please remove it.